<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>268</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>455</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>gmail-plasmoid Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAccounts">
      <attribute name="title">
       <string>Accounts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="btnAdd">
             <property name="text">
              <string>&amp;Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRemove">
             <property name="text">
              <string>&amp;Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMoveUp">
             <property name="text">
              <string>Move &amp;Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMoveDown">
             <property name="text">
              <string>Move &amp;Down</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="KListWidget" name="listAccounts">
           <property name="toolTip">
            <string>Double-click an account to modify it</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAppearance">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Icons:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Text color:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="KColorCombo" name="colColor">
           <property name="toolTip">
            <string>Select the text color for the unread email indicator</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Text font:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Text size:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="intTextSize">
           <property name="toolTip">
            <string>Select the text size, as a percentage of the height of the widget, for the unread email indicator</string>
           </property>
           <property name="suffix">
            <string notr="true">%</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="KFontComboBox" name="fonTextFont">
           <property name="toolTip">
            <string>Select the font for the unread email indicator</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>No Unread Email</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="1" column="0">
               <widget class="KIconButton" name="icoIconNone">
                <property name="toolTip">
                 <string>Select background icon used when there are no unread messages</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="KComboBox" name="cmbDefaultIconNone">
                <property name="toolTip">
                 <string>Use this dropdown if you wish to use one of the default icons</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Unread Email</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <widget class="KIconButton" name="icoIcon">
                <property name="toolTip">
                 <string>Select background icon used when there are unread messages</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="KComboBox" name="cmbDefaultIcon">
                <property name="toolTip">
                 <string>Use this dropdown if you wish to use one of the default icons</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMisc">
      <attribute name="title">
       <string>Misc.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Polling interval:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="intPollingInterval">
           <property name="toolTip">
            <string>Polling interval in minutes</string>
           </property>
           <property name="suffix">
            <string> min(s)</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Command:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="txtCommand">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Command to run when icon clicked (&quot;%u&quot; will be replaced with the url of the first gmail account configured)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCommandBrowse">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Check network:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="chkNetwork">
           <property name="toolTip">
            <string>Check status of network before checking for email (turn this off if you have problems with network detection).</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Debug output:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="chkDebug">
           <property name="toolTip">
            <string>Output debugging details to console</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Fetching mechanism:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="KComboBox" name="cmbFetchMechanism">
           <property name="toolTip">
            <string>Select the mechanism used to fetch the atom feed (note that &quot;python&quot; is the default, but it does not support authenticating proxies).</string>
           </property>
           <item>
            <property name="text">
             <string>Python (default)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>KDE (use KDE internet settings)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KFontComboBox</class>
   <extends>KComboBox</extends>
   <header>kfontcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>listAccounts</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnMoveUp</tabstop>
  <tabstop>btnMoveDown</tabstop>
  <tabstop>icoIconNone</tabstop>
  <tabstop>cmbDefaultIconNone</tabstop>
  <tabstop>icoIcon</tabstop>
  <tabstop>cmbDefaultIcon</tabstop>
  <tabstop>fonTextFont</tabstop>
  <tabstop>intTextSize</tabstop>
  <tabstop>colColor</tabstop>
  <tabstop>txtCommand</tabstop>
  <tabstop>btnCommandBrowse</tabstop>
  <tabstop>intPollingInterval</tabstop>
  <tabstop>cmbFetchMechanism</tabstop>
  <tabstop>chkNetwork</tabstop>
  <tabstop>chkDebug</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
